spiketools/stats/shuffle.py

```diff
@@ -26,6 +26,8 @@ def shuffle_spikes(spikes, approach, n_shuffles=1000, **kwargs):
         See shuffle sub-functions for details.
     n_shuffles : int, optional, default: 1000
         The number of shuffles to create.
+    start_time : float, optional
```

Comment:
I am just wondering if there is a context in which it's also useful to have `time_range` instead of `start_time`?

Reply:
Yeh - I thought a bit about this, but the answer isn't super obvious. In the conversion, sometimes `start_time` vs `time_range` makes sense, and so I added one or the other. In the shuffles, partly to try and make it consistent across function, I defaulted to `start_time` across the board - however some could perhaps use time_range instead. 

In my very brief thinking on this, I didn't think of a clear example where using time_range instead would make a clear difference to the output - did you have an example of this in mind?